chat/message.go

```diff
@@ -23,6 +23,9 @@ type Message struct {
 	// Text of the message to send.
 	Text string `json:"text,omitempty"`
 
+	// Markdown enables Markdown support.
+	Markdown bool `json:"mrkdwn,omitempty"`
```

Comment:
I disagree. Attachment has `mrkdwn_in` for its `Markdown` field, so it makes sense for this to remain consistent with that.